### PR TITLE
U4-8719 - bugfix the redirect dashboard

### PR DESCRIPTION
**Pull request by zpqrtbnk**

### CODE REVIEW

File: src/Umbraco.Web/Umbraco.Web.csproj [hunk 1]
```diff
@@ -209,6 +209,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Microsoft.AspNet.WebApi.Client.5.2.3\lib\net45\System.Net.Http.Formatting.dll</HintPath>
     </Reference>
+    <Reference Include="System.Runtime">
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web/Umbraco.Web.csproj]: Any reason why this is suddenly necessary?

**Comment by nul800sebastiaan** [comment]
@zpqrtbnk Can you check if https://github.com/umbraco/Umbraco-CMS/pull/1457/commits/8d291efedd90d0aa43048caff0de8b31cd7dd96c is okay? Just moved the admin check to a central extension method to make sure we don't repeat ourselves.

@simonbusborg Can you check the localization updates in https://github.com/umbraco/Umbraco-CMS/pull/1457/commits/02cef438159be33dde29365375112c5c936abd35 please? Tested it all and it works.

Just FYI: I think that whenever we expose something to the content section people should be able to localize all of it.

Then there's one more thing, if a content editor just opens the backoffice, this is now the first dashboard you see and it might be confusing without context. We should have a "help" button that explains what this dashboard is for.

image

**Comment by zpqrtbnk** [comment]
Reviewed the IsAdmin thing and all looks OK - just realized I had pushed a "tmp" commit in that PR, how ugly - the System.Runtime reference should never have been pushed, good that you cleared it - all looking good to me.

Do we need a full "help" button or just a few lines of text describing what the whole thing is, left to the enable/disable button & search box?

**Comment by simonbusborg** [comment]
Reviewed and tested localization and all looks good 👍

**Pull request merged**